src/promptflow-devkit/promptflow/_sdk/_utilities/general_utils.py

```diff
@@ -759,6 +759,11 @@ def _generate_connections_dir():
 
 
 # This function is used by extension to generate the connection files every time collect tools.
+# MUST READ!!!
```

Comment:
Using comments to avoid the the misbehavior is only for shortterm and a little hack. If we do need to refer this in another package, we need to make this as an explicit interface instead of referring internal code.